examples/14-elasticsearch/score.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: score.dev/v1b1
+
+metadata:
+  name: example
+
+containers:
+  hello:
+    image: busybox
+    command: ["/bin/sh", "-c"]
+    args:
+      [
+        "echo",
+        "curl --cacert /tmp/ca.crt -u $ECS_USER:$ECS_PASS https://$ECS_HOST:$ECS_PORT",
```

Comment:
In this example, wehere does the ca.crt come from? There are no volume mounts? Does it require a manual step?

Reply:
The `ca.crt` certificate is located in the container in the `/usr/share/elasticsearch/config/certs/ca/` directory and must be copied from there manually. A note about this is displayed in the info block after the docker-compose.yaml file is created in the console.

```shell
INFO: elasticsearch.default#example.ecs: elasticsearch.default#example.ecs: To connect to elasticsearch:
 download certificate from container per command like:
        docker cp [CONTAINER-NAME]:/usr/share/elasticsearch/config/certs/ca/ca.crt /tmp/
 and than check connection per culr like:
        curl --cacert /tmp/ca.crt -u elastic:lj6LSY7XvqY4Ibfe https://localhost:9200
DEBUG: Provisioned resource 'elasticsearch.default#example.ecs'
```

I can't find a helpful approach for the example file.
Do you have a better suggestion than copying the certificate manually?

Reply:
Ok I see. Ok that's probably alright then. Unless we just turn TLS off completely

Reply:
The [article](https://www.elastic.co/blog/getting-started-with-the-elastic-stack-and-docker-compose/) explicitly states that security should not be deactivated.

> **Setup and Elasticsearch node**
> 
> One of the first bits of trouble that's often run into when getting started is security configuration. As of 8.0, security is enabled by default. Therefore, we'll need to make sure we have the certificate CA setup correctly by utilizing a "setup" node to establish the certificates. Having [security enabled is a recommended practice](https://www.elastic.co/guide/en/elasticsearch/reference/current/secure-cluster.html) and should not be disabled, even in POC environments.

That's why I think we should leave it active.